src/component/feedback/FeedbackCES/FeedbackCESScore.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import produce from 'immer';
+import { useStyles } from 'component/feedback/FeedbackCES/FeedbackCESScore.styles';
+import { IFeedbackCESForm } from 'component/feedback/FeedbackCES/FeedbackCESForm';
+
+interface IFeedbackCESScoreProps {
+    form: Partial<IFeedbackCESForm>;
+    setForm: React.Dispatch<React.SetStateAction<Partial<IFeedbackCESForm>>>;
+}
+
+export const FeedbackCESScore = ({ form, setForm }: IFeedbackCESScoreProps) => {
+    const styles = useStyles();
+
+    const onScoreChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        event.persist();
```

Comment:
What version of React are we using? [According to the docs, `event.persist` doesn't do anything from React 17 an onwards](https://reactjs.org/docs/legacy-event-pooling.html), so I'm not sure this is necessary.

Reply:
I mean, I suppose I could check the package.json file, but why spend 30 seconds doing that when you can spend way longer reading about details 😅  But yeah, the package.json says v17.0.2, so I don't think we need this anymore 😄 

![image](https://user-images.githubusercontent.com/17786332/160596055-9aabbc93-84ee-45d8-918c-2902c20de50d.png)

Reply:
Nice catch, I thought that was a v18 thing!